### PR TITLE
Add condensed strokes for "travelled", "traveller", "immortal", "bitterness".

### PR DESCRIPTION
**Pull request by paulfioravanti**
This PR proposes the following condensed strokes, and their usage in the Gutenberg dictionary:

- Following on from #270:
  - An Australian spelling of "travelled" (`TRAFL/*LD`) that works with straight Plover
  - An Australian spelling of "traveller" (`TRA*FL/HRER`) that works with straight Plover
- An outline for "immortal" that does not require an asterisk in the final stroke: `EUPL/PHORT/AL`
- An outline for "bitterness" that leverages the `PWEURT` outline for "bitter" and the `-PBS` "ness" suffix

If any of the strokes are not considered appropriate to use in Typey Type/Gutenberg, then I'm happy to take them out of the Gutenberg dictionary.

### CODE REVIEW

File: dictionaries/condensed-strokes.json [hunk 1]
```diff
@@ -1,6 +1,10 @@
 {
+"TRA*FL/HRER": "traveller",
```
Comment (didoesdigital) [inline dictionaries/condensed-strokes.json:2]: @paulfioravanti could this be `TRAFL/HRER`?

File: dictionaries/condensed-strokes.json [hunk 2]
```diff
@@ -1,6 +1,10 @@
...
+"PWEURT/-PBS": "bitterness",
+"EUPL/PHORT/AL": "immortal",
```
Comment (didoesdigital) [inline dictionaries/condensed-strokes.json:4]: As per https://github.com/didoesdigital/steno-dictionaries/issues/288#issuecomment-609504922, I would not add this entry using the phonetic `/AL` but instead stick to the entry using a prefix and suffix stroke: `"EUPL/PHORT/A*L": "immortal",`

File: dictionaries/top-10000-project-gutenberg-words.json [hunk 3]
```diff
@@ -4172,7 +4172,7 @@
 "KRORPB/-S": "corners",
 "PHAG": "magazine",
 "HROUD/HREU": "loudly",
-"PWEUT/*ERPBS": "bitterness",
+"PWEURT/-PBS": "bitterness",
```
Comment (didoesdigital) [inline dictionaries/top-10000-project-gutenberg-words.json:4175]: Nice one 👏

File: dictionaries/top-10000-project-gutenberg-words.json [hunk 4]
```diff
@@ -4377,7 +4377,7 @@
 "PW*P/KR*P": "BC",
 "AEUB/HAPL": "Abraham",
 "THAUFL": "thoughtful",
-"TRAFL/HRER/A*U": "traveller",
+"TRA*FL/HRER": "traveller",
```
Comment (didoesdigital) [inline dictionaries/top-10000-project-gutenberg-words.json:4380]: As above

File: dictionaries/condensed-strokes.json (hunk 1 above)
Comment (paulfioravanti) [reply dictionaries/condensed-strokes.json:2]: `TRAFL/HRER` is already in the dictionaries as "traveler" with one "l". In order to get English spelling, it needed the asterisk.

File: dictionaries/condensed-strokes.json (hunk 2 above)
Comment (paulfioravanti) [reply dictionaries/condensed-strokes.json]: Yep, understood. Fixed.

File: dictionaries/top-10000-project-gutenberg-words.json (hunk 4 above)
Comment (paulfioravanti) [reply dictionaries/top-10000-project-gutenberg-words.json:4380]: Yep. As per [this comment](https://github.com/didoesdigital/steno-dictionaries/pull/276#discussion_r406086750), `TRAFL/HRER` is already in the dictionaries as "traveler".

**Pull request merged**